src/main/webapp/js/contextualcomments.es6

```diff
@@ -18,7 +18,7 @@ function prepareComments() {
     if (typeof richTextEditorBuilder !== 'undefined') {
         /* eslint-disable camelcase */ // The property names are determined by external library (tinymce)
         richTextEditorBuilder.initEditor('#commenttext', {
-            inline: true,
+            inline: false,
```

Comment:
@damithc Apologies. Upon closer review of the [tinyMCE documentation](https://www.tinymce.com/docs/configure/editor-appearance/#inline) and trying it out on the local server, you are correct. Omitting this parameter would allow the editor to default to `false.` Would you recommend new commits that do so?